README.md

```diff
@@ -31,3 +31,50 @@ explained in the config file.
 
 * How to run:
 `node scripts/fetchODCDataFiles.js https://data.ontario.ca/dataset/covid-19-assessment-centre-locations ODC`
+
+### Diff/merge 2 or 3 datasets
+
+* Script: scripts/test_daff.js
+
+* Goal: This script runs a diffing algorithm and produces html that shows a visulization of the diff and optionally an
+updated local data set that is a result of mergeing in the diff results into the local file. The visuliztion can be seen
+by running the produced html file in a browser. This script takes three arguments maximum but needs at least two arugments
```

Comment:
Can you expand on the difference of taking 2 or 3 arguments? If there's a daff doc about this, it's helpful to add the doc link here.

Reply:
@TeddTech and I chatted about how to better improve the poc_daff.js script:

Right now, this script does:
1. calculate the similarity between columns;
2. generate the column match metadata file;
3. replace the column names in the data files to match;
4. run the diff2 or diff3 to generate the diffing html.

Since the point 4 has already been covered by test_diff2.js and test_diff3.js. This script can be modified to only perform the first 3 parts.

Changes to "test_daff.js" include:
1. rename the script name to "calcColumnSimilarity.js";
2. move all daff scripts into "/daff" directory;
3. an example to run this script from the root directory:
```
node daff/calcColumnSimilarity.js daff/data/weCount/ODC1.csv daff/data/weCount/WeCount1.csv daff/data/weCount/metadata_ODC1_WeCount1.json
```
The first 2 arguments are location of input data files. The last argument is the location of the output data file.
4. The "/data" directory can be removed.